pyproject.toml

```diff
@@ -168,6 +168,13 @@ dataloaders=[
     "pytube",
     "sentence-transformers",
     "unstructured",
+    "jq",
+    "requests",
+    "google-api-python-client",
+    "google-auth",
+    "google-auth-oauthlib",
+    "google-auth-httplib2",
+    "google-api-core",
 ]
```

Comment:
Can we please create a new `extra` called `gmail` and add all these loaders there as well?

Reply:
sure